TACC/Core-Portal
# task/FP-628 -- Allow partial match of host resources

## Overview: ##

'login1.frontera.tacc.utexas.edu' should be valid for 'frontera.tacc.utexas.edu' allocation hosts.

## PR Status: ##


## Related Jira tickets: ##

* [FP-628](https://jira.tacc.utexas.edu/browse/FP-628)

## Summary of Changes: ##

- Allow host resource partial matches when rendering AppSchemaForm
- Unit tests for AppSchemaForm

## Testing Steps: ##
1. Create a private execution system for 'login1.frontera.tacc.utexas.edu'
2. Create a private app tied to this execution system
3. Make sure you can launch it

## Notes: ##

I have not followed these testing procedures yet. May have to wait until next release for this one.
